src/sql/magic.py

```diff
@@ -367,11 +373,34 @@ def interactive_execute_wrapper(**kwargs):
             alias=args.alias,
         )
         payload["connection_info"] = conn._get_curr_sqlalchemy_connection_info()
-        if args.persist:
+        if args.persist_replace and args.append:
+            raise exceptions.ValueError(
+                """You cannot simultaneously replace and append data to a dataframe;
+                  please choose to utilize either one or the other."""
+            )
+        if args.persist and args.persist_replace:
+            print("Please use either --persist or --persist-replace")
```

Comment:
this should raise a UsageError - please add a unit test as well

Reply:
[In earlier discussion] (https://github.com/ploomber/jupysql/pull/440#discussion_r1177515061) they thought we can still apply the persist_replace and throw the warning

What do you think? Should we just raise the UsageError with `Please use either --persist or --persist-replace` ?

Reply:
Fix!